builder/lxd/builder.go

```diff
@@ -61,8 +61,13 @@ func (b *Builder) Run(ctx context.Context, ui packersdk.Ui, hook packersdk.Hook)
 		return nil, rawErr.(error)
 	}
 
+	id := "0"
+	if !b.config.SkipPublish {
+		id = state.Get("imageFingerprint").(string)
+	}
```

Comment:
If the id can be just an empty string, then you can do
```suggestion
	id, _ := state.Get("imageFingerprint").(string)
```
See example: https://go.dev/play/p/3wLBxPSyFGH

Reply:
thanks! I will try your suggestion.